azavea/raster-foundry
# New project routes (no merge)

## Overview

This PR will not be merged, but I want to get some feedback on the route setup before I get too far into implementing stuff

* New Project routes
* Collapse navbar when in project edit view

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

## Testing Instructions

* Look at the designs here: https://drive.google.com/file/d/0B34sFvhrpo5COHdycm9DTkNtWk0/view
* Go over the routes and make sure that they make sense

Connects #1437 
Connect #1438
